### DISCUSSION TITLE
Files (using native file manager) should have an 'alt text' field

### DISCUSSION THREAD

**Discussion by agun** [ExpressionEngine/ExpressionEngine#2879] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When adding/editing a new file via the native EE file manager (EE7), you can add in a Title, Description, Credit and Location. This is great, but it would be nice to be able to add in a specific 'Alt text' field to help with accessibility.

It's confusing for authors to use the 'title' or 'description' field for alt text (they have told me as much). We often also want to have a different Title to Alt text on an image.

**Describe the solution you'd like**
A new Alt text field for all files in the native file manager. 

**Describe alternatives you've considered**
The ability to add in custom fields for files would be a good alternative ... however I suspect just adding in the alt title is easier!

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by jaredchesebro** [comment]
I agree that the current field set is not very intuitive. Now that it's possible to edit file meta directly within an entry I think it's time to either update the field names or allow admins the ability to add new fields to the file manager.
